x/feegrant/types/expiration.go

```diff
@@ -63,7 +56,7 @@ func (e ExpiresAt) IsExpired(t time.Time, h int64) bool {
 
 // IsCompatible returns true iff the two use the same units.
 // If false, they cannot be added.
-func (e ExpiresAt) IsCompatible(d Duration) bool {
+func (e ExpiresAt) IsCompatible(d *Duration) bool {
```

Comment:
Why do we need to accept a pointer?

Reply:
I think this can be eliminated. Will update